frontend/src/components/utils/table/TableBody.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Columns } from "./TableHeader";
+
+const Row = ({
+    columns,
+    data,
+    index
+}: {
+    columns: Columns;
+    data: object;
+    index: number;
+}) => (
+    <tr role="row" className={index % 2 == 0 ? "even" : "odd"}>
```

Comment:
In bootstrap there is a way to have colour alternance without adding specific classes to HTML

Reply:
I don't know, I saw this on Tabler.io's website